File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/domain/entity/discovery/SupplierInfo.kt
package com.sudo248.sudoo.domain.entity.discovery

import com.sudo248.sudoo.domain.entity.user.Address

data class SupplierInfo(
    val supplierId: String,
    val name: String,
    val avatar: String,
    val brand: String,
    val contactUrl: String,
    val rate: Float,
    val address: Address
)

File: client/web/sudoo/lib/domain/model/discovery/supplier_info.dart
import 'package:sudoo/domain/model/discovery/product.dart';

class SupplierInfo {
  final String supplierId;
  final String name;
  final String avatar;
  final String brand;
  final String contactUrl;
  final int? totalProducts;
  final double? rate;
  final List<Product>? products;

  const SupplierInfo(
    this.supplierId,
    this.name,
    this.avatar,
    this.brand,
    this.contactUrl,
    this.totalProducts,
    this.rate, {
    this.products,
  });
}
